src/test/java/fr/avenirsesr/portfolio/testutils/BddLogger.java

```diff
@@ -0,0 +1,22 @@
+package fr.avenirsesr.portfolio.testutils;
+
+public final class BddLogger {
+
+  private static void log(String prefix, String message) {
+    System.out.println(prefix + " " + message);
+  }
+
+  private BddLogger() {}
```

Comment:
C'est du détail, mais grâce à Lombok tu peux utiliser l'annotation @NoArgsConstructor au dessus de ta class, il génère automatiquement un constructor private et vide, comme ça plus besoin de le déclarer 
```suggestion
@NoArgsConstructor
public final class BddLogger {

  private static void log(String prefix, String message) {
    System.out.println(prefix + " " + message);
  }

```

Reply:
Ha merci je connaissais pas, je fais ça !